doc/UsersGuide/BuildingRunningTesting/ContainerQuickstart.rst

```diff
@@ -78,6 +78,8 @@ On most Level 1 systems, a container named ``ubuntu22.04-intel-ue-1.6.0-srw-dev.
      - /glade/work/epicufsrt/contrib/containers
    * - Gaea-C5 [#fn]_
      - /gpfs/f5/epic/world-shared/containers
+   * - Gaea-C6 [#fn]_
+     - /gpfs/f6/bil-fire8/world-shared/containers
```

Comment:
In line 136, there is a build statement that should be updated for the v3.0.0 release.